Bento/Renderable/ComponentLifecycleAware.swift

```diff
@@ -7,3 +7,35 @@ public protocol ViewLifecycleAware {
     func willDisplayView()
     func didEndDisplayingView()
 }
+
+final class LifecycleComponent<Base: Renderable>: AnyRenderableBox<Base>, ComponentLifecycleAware where Base.View: UIView {
+    private let source: AnyRenderableBox<Base>
+    private let _willDisplayView: (() -> Void)?
```

Comment:
We don't use `_` as prefix and there's no collision between `willDisplayItem` and `willDisplayView`.